packages/pages/src/scaffold/modules/generate.ts

```diff
@@ -92,7 +98,11 @@ const validateModuleName = (
   if (fs.existsSync(modulePath)) {
     return false;
   }
-  return /^[a-zA-Z]+$/.test(moduleName.charAt(0));
+  return (
```

Comment:
Does this happen after we set it?

Reply:
checks validation while user is trying to input via prompts. prompts library rejects user input if it doesn't pass